go/cmd/pserver/pserver.go

```diff
@@ -20,6 +20,8 @@ func main() {
 		"comma separated endpoint string for pserver to connect to etcd")
 	etcdTimeout := flag.Int("etcd-timeout", 5, "timeout for etcd calls")
 	numPservers := flag.Int("num-pservers", 1, "total pserver count in a training job")
+	checkpointPath := flag.String("checkpoint-path", "/checkpoints/", "save checkpoint path")
+	checkpointInterval := flag.Int("checkpoint-interval", 10, "save checkpoint per interval seconds")
```

Comment:
yeah, fix interval is not proper for every training job. Time consumed always determined by training data amount. `Round` count may be better here.
Change it to 10 min(600seconds)